libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java

```diff
@@ -187,8 +190,9 @@ private void updateArrivalTime(RouteProgress routeProgress) {
     double durationRemaining = routeProgress.durationRemaining();
     int timeFormatType = options.timeFormatType();
     String arrivalTime = formatTime(time, durationRemaining, timeFormatType, isTwentyFourHourFormat);
-    collapsedNotificationRemoteViews.setTextViewText(R.id.notificationArrivalText, arrivalTime);
-    expandedNotificationRemoteViews.setTextViewText(R.id.notificationArrivalText, arrivalTime);
+    String formattedArrivalText = String.format(Locale.getDefault(), etaFormat, arrivalTime);
```

Comment:
Would it be better to get the `Configuration` `Locale` using `LocaleUtils` here?  We can do this during `initialize`.  I believe we are doing this everywhere else within the SDK, so it would be great to stay consistent.